src/xdocs/google_style.xml

```diff
@@ -1615,7 +1615,18 @@
                   <a href="styleguides/google-java-style-20220203/javaguide.html#s5.1-identifier-names">
                     5.1 Rules common to all identifiers</a>
                 </td>
-                <td>"5.2 Rules of identifier type" already includes this rule.
+                <td>
+                  <span class="wrapper inline">
+                    <img src="images/ok_green.png" alt="" />
+                  </span>
+                  <a href="checks/naming/catchparametername.html#CatchParameterName">CatchParameterName</a>
```

Comment:
Please add comment `all other types of identifiers has special  section below `
It is not obvious why catchpatamernsme is covering this section.